api/conftest.py

```diff
@@ -63,6 +67,51 @@
 )
 from users.models import FFAdminUser, UserPermissionGroup
 
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--ci",
+        action="store_true",
+        default=False,
+        help="Enable CI mode",
+    )
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config: pytest.Config) -> None:
+    if (
+        config.option.ci
+        and config.option.dist != "no"
+        and not hasattr(config, "workerinput")
+    ):
+        with config.stash[blocking_manager_key].unblock():
+            setup_databases(
+                verbosity=config.option.verbose,
+                interactive=False,
+                parallel=config.option.numprocesses,
+            )
+
+
+@pytest.fixture(scope="session")
+def django_db_setup(request: pytest.FixtureRequest) -> None:
+    if (
+        request.config.option.ci
+        and (xdist_worker_id_suffix := get_xdist_worker_id(request)[2:]).isnumeric()
```

Comment:
It's not obvious why 

```python
get_xdist_worker_id(request)[2:]
```

That part of the logic ends with a `[2:]`. Can you add a comment?

Reply:
Added.